### PR TITLE
chore: cleanup for create command

### PR DESCRIPTION
**Pull request by wtrocki**
Wrong arguments were passed. 
I do not know what openshift token (missed that during previous review)

### CODE REVIEW

File: plugins/modules/create_kafka.py [hunk 1]
```diff
@@ -75,13 +68,9 @@
...
     redhat.rhoask.create_kafka:
       name: "kafka_name"
-      instance_type: "x1"
-      billing_model: "standard"
+      plan: "developer.x1"
       cloud_provider: "aws"
       region: "us-east-1"
-      plan: "developer.x1"
-      billing_cloud_account_id: "123456789"
-      openshift_offline_token: "OPENSHIFT_CLUSTER_MANAGER_API_OFFLINE_TOKEN"
```
Comment (wtrocki) [inline plugins/modules/create_kafka.py:84]: It that would be our token to work then we picked the most confusing name. 
Even as insider I have no clue what this token is. I would call it refreshtoken or offline_token at most

Comment (dimakis) [reply plugins/modules/create_kafka.py:84]: https://console.redhat.com/openshift/token/show# ...

Comment (wtrocki) [reply plugins/modules/create_kafka.py:84]: Long story short. We were supposed to get our own token page but this never happened. 
OpenShift Cluster manager is completely different product. We are using "their" token page to get refresh token as there is no other way. 

Proper names are:

- RefreshToken
- OfflineToken

Then doc should tell how to get it (and that will likely to change).

There is also other token page: https://console.redhat.com/ansible/automation-hub/token
Maybe we could use that token instead (it has ansible in name by coincidence)

File: plugins/modules/create_kafka.py [hunk 2]
```diff
@@ -54,14 +54,7 @@
         description: Billing model for the Kafka instance.
         required: true 
         type: str
-    instance_type:
```
Comment (dimakis) [inline plugins/modules/create_kafka.py:57]: why remove the instance type?

Comment (wtrocki) [reply plugins/modules/create_kafka.py:57]: To long for comment on github. We need to do some small AMS training in the team.

**Pull request closed without merging**